src/rpc/util.h

```diff
@@ -137,7 +137,6 @@ struct RPCResult {
     explicit RPCResult(std::string result)
         : m_cond{}, m_result{std::move(result)}
     {
-        assert(!m_result.empty());
```

Comment:
should revert this and use `RPCResults{}` instead of `RPCResult{""}`, see 9a23282656

Reply:
Thanks, good point - I was getting the assert triggered but haven't realized an empty plural struct is what's needed to prevent it from going off.